script/M1_Deploy.s.sol

```diff
@@ -77,6 +77,8 @@ contract Deployer_M1 is Script, Test {
 
     // IMMUTABLES TO SET
     uint256 REQUIRED_BALANCE_WEI;
+    uint256 MAX_VALIDATOR_BALANCE_GWEI;
+    uint256 EFFECTIVE_RESTAKED_BALANCE_OFFSET;
```

Comment:
should this also have `GWEI` in the name? seems like it's in gwei?

Reply:
fixed.